sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -1681,8 +1681,10 @@ object GpuOverrides extends Logging {
     expr[KnownFloatingPointNormalized](
       "Tag to prevent redundant normalization",
       ExprChecks.unaryProjectInputMatchesOutput(
-        TypeSig.DOUBLE + TypeSig.FLOAT,
-        TypeSig.DOUBLE + TypeSig.FLOAT),
+        TypeSig.DOUBLE + TypeSig.FLOAT + TypeSig.ARRAY.nested(
+          TypeSig.commonCudfTypes + TypeSig.DECIMAL_128 + TypeSig.NULL),
+        TypeSig.DOUBLE + TypeSig.FLOAT + TypeSig.ARRAY.nested(
+          TypeSig.commonCudfTypes + TypeSig.DECIMAL_128 + TypeSig.NULL)),
```

Comment:
I didn't want to open it wider than it was needed thinking we can always types later. Are you saying we should just open it for `TypeSig.all`?

Reply:
Yes.  The implementation is a pass through regardless of input type, so IMO we want this to be on the GPU for any type the operation supports on the CPU.  Put another way, do we really want this operation to be the only operation not on the GPU because we forgot to update one of the type signatures?